crates/blockchain/mempool.rs

```diff
@@ -168,6 +168,9 @@ fn validate_transaction(tx: &Transaction, store: Store) -> Result<(), MempoolErr
     let header_no = store
         .get_latest_block_number()?
         .ok_or(MempoolError::NoBlockHeaderError)?;
+    let block_header = store
+        .get_block_header(header_no)?
+        .ok_or(MempoolError::NoBlockHeaderError)?;
     let header = store
         .get_block_header(header_no)?
         .ok_or(MempoolError::NoBlockHeaderError)?;
```

Comment:
This is duplicated

Reply:
Removed here [9c20b3a](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1125/commits/9c20b3a6c620f1693dc0c8956c82cb92f6f23b25)